spec/config/initializers/job_configurations_spec.rb

```diff
@@ -0,0 +1,13 @@
+# cron.each do |key, value|
+# [12] pry(main)*   value[:class].constantize.new
+# [12] pry(main)* end
+
+require 'rails_helper'
+
+describe 'job_configurations' do
+  it 'has valid cron jobs' do
+    Rails.application.config.good_job.cron.each do |key, config|
+      config[:class].constantize.new
+    end
```

Comment:
1. what if we use aggregate_failures?
2. what if we check the superclass?
3. maybe add other assertions?
```suggestion
    aggregate_failures do
      Rails.application.config.good_job.cron.each do |key, config|
        expect(config[:class].constantize).to be_kind_of(ApplicationJob)
        expect(Fugit.parse(config[:cron])).to_not be_nil
      end
    end
```